internal/cmd/launcher.go

```diff
@@ -283,31 +282,32 @@ func analyze(gs *state.GlobalState, args []string) (k6deps.Dependencies, error)
 		Manifest:  k6deps.Source{Ignore: true},
 	}
 
-	if len(args) == 0 {
-		gs.Logger.
-			Debug("The command did not receive an input script.")
-		return nil, errScriptNotFound
+	if len(args) < 1 {
+		return nil, fmt.Errorf("the invoked command needs a file path or pass the test via Stdin")
```

Comment:
nit, assuming stdin input works in a similar fashion to `k6 run`.

```suggestion
		return nil, fmt.Errorf("missing test input: provide a file path as argument or pass script content via stdin using '-'")
```